rotkehlchen/api/rest.py

```diff
@@ -280,6 +285,49 @@ def inner(rest_api: 'RestAPI', async_query: bool = False, **kwargs: Any) -> Resp
     return wrapper
 
 
+def check_cached_image(image_path: Path, match_header: Optional[str]) -> Optional[Response]:
```

Comment:
I think the name of this function conflicts with what it does. I might be wrong though.